src/containers/Projects/index.js

```diff
@@ -98,18 +123,34 @@ export class Projects extends Component {
               </div>
 
             </Sticky>
-
-            <div>
-              <ProjectList
-                data={projects}
-                valueProp="id"
-                childrenProp="projects"
-                primaryTextProp="name"
-                secondaryTextProp="description"
-                clickHandler={value => dispatch(push(`/project/${value}`))}
-                {...theme}
-              />
-            </div>
+            {!!groups.length &&
+              <div>
+                <GroupList
+                  data={groups}
+                  valueProp="id"
+                  childrenProp="projects"
+                  primaryTextProp="name"
+                  secondaryTextProp="description"
+                  clickHandler={value => this.clickHandler(value)}
+                  {...theme}
+                />
+              </div>
+            }
+            {!!projects.length &&
+              <div>
+                <ProjectList
+                  data={projects}
+                  valueProp="id"
+                  childrenProp="projects"
+                  primaryTextProp="name"
+                  secondaryTextProp="description"
+                  clickHandler={value => dispatch(push(`/project/${value}`))}
```

Comment:
Same here - function created every time component renders.

I would add helper function to `routes`, e.g.
 ```
export const (repoName) => buildProjectRoute {
   return `/project/${repoName}`
```
to build the route value, in case we change some routes we wouldn't break everything and the change will be reflected in all components(There some helpers already added)